test/integration/auth/podsecurity_test.go

```diff
@@ -39,10 +42,33 @@ func TestPodSecurity(t *testing.T) {
 	opts := podsecuritytest.Options{
 		ClientConfig: server.ClientConfig,
 
+		// Don't pass in feature-gate info, so all testcases run
+
 		// TODO
 		ExemptClient:         nil,
 		ExemptNamespaces:     []string{},
 		ExemptRuntimeClasses: []string{},
 	}
 	podsecuritytest.Run(t, opts)
 }
+
+// TestPodSecurityGAOnly ensures policies pass with only GA features enabled
+func TestPodSecurityGAOnly(t *testing.T) {
+	// Disable all alpha and beta features
+	defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, "AllAlpha", false)()
+	defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, "AllBeta", false)()
```

Comment:
/hold

this doesn't work right to restore previous state exactly

Reply:
reworked to iterate and call SetFeatureGateDuringTest on alpha/beta features individually